nebula/tests/transcript/test_queue_worker.py

```
@@ -110,3 +110,204 @@ async def fake_light_phase(job_id, req, transcription, video_path, audio_path, u
+
+    assert "job1" in remaining_jobs
+    assert "job2" not in remaining_jobs
+    assert "job3" in remaining_jobs
+
+
+@pytest.mark.anyio
+async def test_cancel_job_during_processing_stops_it(monkeypatch):
+    """Test that cancelling a job during processing stops it and cleans up."""
+    monkeypatch.setattr(qw, "_job_queue", asyncio.Queue())
+
+    cancel_event = asyncio.Event()
```

Ruff (0.14.4)

Local variable `cancel_event` is assigned to but never used

Remove assignment to unused variable `cancel_event`

(F841)